contracts/interfaces/IMultisigBase.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+interface IMultisigBase {
```

Comment:
Multisig contract should likely be moved to gmp sdk too, but i'm ok with getting it merged first for governance feature completeness